### PR TITLE
bump: Node 18 LTS

### PR DESCRIPTION
**Pull request by johanandren**
Just trying out to see if it breaks things

### CODE REVIEW

**Comment by pvlugter** [comment]
👍🏼 seems good, and makes sense to do this bump and release with a breaking version bump for the SDK

**Comment by ennru** [comment]
Would be good to go to Node 18 now.

File: npm-js/create-kalix-entity/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "author": "Lightbend Inc",
   "license": "Copyright (c) Lightbend Inc. 2021",
   "engines": {
-    "node": ">=14.0.0",
+    "node": ">=18.13.0",
```
Comment (pvlugter) [inline npm-js/create-kalix-entity/package.json:10]: Should this be `>=18.0.0` or `~18`?

**Review by pvlugter — APPROVED** [review]
LGTM

File: npm-js/create-kalix-entity/package.json (hunk 1 above)
Comment (johanandren) [reply npm-js/create-kalix-entity/package.json:10]: Not sure how JS people usually do it. I guess ~18 is fine?

Comment (pvlugter) [reply npm-js/create-kalix-entity/package.json:10]: Sounds good. That's been used in the other places.

Comment (johanandren) [reply npm-js/create-kalix-entity/package.json]: Done in [213245](https://github.com/lightbend/kalix-javascript-sdk/pull/459/commits/221324592055a1e63007d337bbcd6ca838792be7) 

We'll see what breaks. :)

**Review by pvlugter — APPROVED** [review]
Still LGTM

**Comment by pvlugter** [comment]
We should also have a version bump for the SDK — 1.1.0 or 2.0.0 for this?

**Comment by pvlugter** [comment]
Will merge now, and we can then keep updating for node 18 minor versions separately.

**Pull request merged**